app/services/articles/analytics_updater.rb

```diff
@@ -0,0 +1,60 @@
+module Articles
+  class AnalyticsUpdater
+    def initialize(user, context = "default")
```

Comment:
I don't see much value in passing `context` currently. It's not used anywhere, and not tested. (correct me if I'm wrong).
If it would be needed, I guess `force = true` would be enough.